### PR TITLE
chore: device account followup

### PR DESCRIPTION
**Pull request by UncleSamtoshi**
- handle already existing account errors during phone validation
- improve copy surrounding log in

### CODE REVIEW

File: app/i18n/en/index.ts [hunk 1]
```diff
@@ -728,6 +728,7 @@ const en: BaseTranslation = {
   PhoneValidationScreen: {
     errorLoggingIn: "Error logging in. Did you use the right code?",
     errorTooManyAttempts: "Too many attempts. Please try again later.",
+    errorCannotUpgradeToExistingAccount: "This phone account already exists. Please log out of your trial account and then log in with your phone number.",
```
Comment (nicolasburtey) [inline app/i18n/en/index.ts:731]: have we settled down on the `trial` account terminology?

Comment (UncleSamtoshi) [reply app/i18n/en/index.ts:731]: Right now yes.

File: app/screens/home-screen/home-screen.tsx [hunk 2]
```diff
@@ -292,9 +293,9 @@ export const HomeScreen: React.FC = () => {
           setIsStablesatModalVisible={setIsStablesatModalVisible}
         />
         {error && (
-          <Text style={styles.error} selectable>
-            {getErrorMessages(error)}
-          </Text>
+          <View style={styles.marginBottonContainer}>
```
Comment (nicolasburtey) [inline app/screens/home-screen/home-screen.tsx:296]:
```suggestion
          <View style={styles.marginButtonContainer}>
```

File: app/screens/home-screen/home-screen.tsx [hunk 3]
```diff
@@ -374,6 +375,9 @@ const useStyles = makeStyles(({ colors }) => ({
     borderRadius: 32,
     borderWidth: 2,
   },
+  marginBottonContainer: {
```
Comment (nicolasburtey) [inline app/screens/home-screen/home-screen.tsx:378]:
```suggestion
  marginButtonContainer: {
```

**Review by nicolasburtey — APPROVED** [review]

**Pull request merged**